.github/workflows/release.yml

```
@@ -16,78 +16,78 @@
-            publish: bun release
-            commit: "ci(changesets): version packages"
-            setupGitUser: false
-          env:
-            NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-            NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-            NPM_CONFIG_PROVENANCE: true
-            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Publish
+        id: changesets
+        uses: changesets/action@v1
+        if: ${{ github.event.inputs.tag == '' }}
```

actionlint (1.7.4)

property "tag" is not defined in object type {}

(expression)